implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -446,3 +448,62 @@ pub(crate) fn replace_hyphen_with_stdin(s: String) -> String {
         s
     }
 }
+
+const EXIT_OK: i32 = 0;
+const EXIT_USAGE: i32 = 2;
+
+fn show_help(help: clap::Error) {
+    paginate(&help.render()).unwrap_or_else(|_| help.print().unwrap());
+
+    let _ = std::io::stdout().lock().flush();
+    let _ = std::io::stderr().lock().flush();
+    process::exit(if help.use_stderr() {
+        EXIT_USAGE
+    } else {
+        EXIT_OK
+    });
```

Comment:
Yeah, I get your point. The reason we try to make things as explicit and documented as possible is because this code (`ockam_command`) is read by people that are contributing to an opensource project for their first time (students, or people playing around with rust, for example), so we try to keep the code as less cryptic as possible, especially around non-obvious logic/assumptions. Of course, we don't want to document code that is self explanatory (as you said, sometimes a good variable name is good enough to explain what's going on).